default.nix

```diff
@@ -56,16 +56,28 @@ cargoLorri.override {
         pkgs.darwin.apple_sdk.frameworks.CoreServices
         pkgs.libiconv
       ];
+      nativeBuildInputs = [ pkgs.installShellFiles ];
 
-      # copy the docs to the $man and $doc outputs
       postInstall = ''
+        # copy the docs to the $man and $doc outputs
+        ${pkgs.scdoc}/bin/scdoc < lorri.scd > lorri.1
         install -Dm644 lorri.1 $man/share/man/man1/lorri.1
         install -Dm644 -t $doc/share/doc/lorri/ \
           README.md \
           CONTRIBUTING.md \
           LICENSE \
           MAINTAINERS.md
         cp -r contrib/ $doc/share/doc/lorri/contrib
+
+        # install shell completions via the internal command
+        mkdir out-completions
+        cd out-completions
+        $out/bin/lorri internal write-shell-completion-scripts
+        echo installing shell completions for ./*
+
+        # || true because installShellCompletion is buggy as hell and I don’t care
```

Comment:
I don't think it is, it just doesn't support elvish. The rest it supports shouldn't have clariying bugs.